apis/bulkupload/bulkserviceapi.yaml

```diff
@@ -75,6 +75,59 @@ paths:
+            $ref: '#/definitions/BulkUploadResponse'
+        '400':
+          description: 'BAD REQUEST. The "bulk upload user" operation failed ! The possible reason for failure is that you may have missed providing input for a mandatory parameter.'
+          schema:
+            $ref: '#/definitions/BulkUploadErrorResponse'
+        '500':
+          description: 'INTERNAL SERVER ERROR ! Looks like something went wrong! These errors are tracked automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.'
+  /user/v2/bulk/upload:
+    post:
+      description: >-
+        This API is associated with upload shadow users in bulk on the Sunbird Platform,shadow user are those who want to migrate from custodian org to non-custodian org.
+           - The endpoint for **Upload Users in Bulk** is `/user/v2/bulk/upload`
+           - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+      summary: Upload Shadow Users in Bulk
+      tags:
+        - Bulk Operation APIs
+      operationId: bulk upload shadow user req1
+      produces:
+        - application/json
+      consumes:
+        - multipart/form-data
+      parameters:
+        - name: user
+          in: formData
+          required: true
+          type: file
+          format: file
+          description: >-
+            This is a .csv file.
+              - Each line of the file is a data record. Each record consists of one or more user fields, separated by commas. Each successfully created user is associated with root org of uploader (org admin)
+            The CSV file format supported is mentioned below:
+              - NAME: Name of user(firstName)
+              - PHONE: Phone number. Either phone or email is mandatory.
+              - EMAIL: Email. Either phone or email is mandatory.
+              - Ext Org id: Organisation ID to which user must be associated(extOrgId).it is not mandatory attribute, if passed user will be assosiated to organisation(RootOrg and Sub-Org).if not then user will be only associated with the org admin root org
+              - Input Status: it is mandatory attribute. this attribute will decide the status(active/inactive) in an organization.
+              - Ext User ID: External ID of user. Either, school ID or external ID is mandatory for user association to organisation.
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'Specify authorization key (format: Bearer api-key) received from administrator when performing given API request.'
+        - name: X-Authenticated-User-Token
+          in: header
+          required: true
+          type: string
+          description: >-
+            Access token of registered user performing given API request.
+
       responses:
         '200':
           description: 'OK ! Successful operation."bulk upload user" operation was successfully executed.'
```

Comment:
success with processid. This process id can be used for further communication.

Reply:
done